test/e2e/providers/azure/azure.go

```diff
@@ -87,10 +81,16 @@ func (p *Provider) newAzureMachineProviderSpec(location string, zone string, win
 			},
 		},
 		PublicIP:             false,
+		PublicLoadBalancer:   publicLoadBalancer,
```

Comment:
> Ensure that Windows 2019 and 2022 VMs without outbound access to the internet are able to be configured on Azure.


@sebsoto  how does this change satisfies this statement?  ([WINC-1189](https://issues.redhat.com/browse/WINC-1189))

Reply:
From https://issues.redhat.com/browse/CORS-3824

https://learn.microsoft.com/en-us/answers/questions/2150029/will-a-public-load-balancer-with-a-public-ipv4-add

By using a public load balancer the Azure changes to outbound access will be a NOOP.

Reply:
I follow